src/renderer/components/ScenePicker.tsx

```diff
@@ -83,10 +83,10 @@ export default class ScenePicker extends React.Component {
                 onChange={this.onSort.bind(this)}
               />
               {this.props.scenes.length > 1 && (
-                <div className="u-random" onClick={this.onRandom.bind(this)}/>
+                <div className="u-random" title="Play a random scene" onClick={this.onRandom.bind(this)}/>
               )}
-              <div className="u-import" onClick={this.props.onImport.bind(this)}/>
-              <div className="u-config" onClick={this.props.onConfig.bind(this)}/>
+              <div className="u-import" title="Import a scene from a file" onClick={this.props.onImport.bind(this)}/>
+              <div className="u-config" title="Preferences" onClick={this.props.onConfig.bind(this)}/>
```

Comment:
OMG ty for adding tooltips to these icons!💕 I looked it up a few times but all I ever found was making your own tooltip component and I knew that just had to be overkill 🤦‍♀️